### PR TITLE
Flask strap

### PR DESCRIPTION
**Pull request by soolaymahn**

### CODE REVIEW

**Comment by Budgetperson** [comment]
Rohit: Let's set up Travis before we merge this

File: app.wsgi [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/python
+import sys
+import logging
+logging.basicConfig(stream=sys.stderr)
+sys.path.insert(0,"/var/www/testbooks/")
+
+from app import app as application
+application.secret_key = 'Add your secret key'
```
Comment (Budgetperson) [inline app.wsgi]: Should we have a real secret key?

**Comment by Budgetperson** [comment]
What else needs to be done to merge?

**Comment by soolaymahn** [comment]
I think it's ready to be merged

**Comment by Budgetperson** [comment]
I'm going to finish all frontend except about then merge.

**Pull request merged**